README.md

```diff
@@ -12,42 +12,77 @@ In your `flake.nix`:
 ```nix
 {
   # Use this repo as the `nixpkgs` URL
-  inputs.nixpkgs = "github:nix-ocaml/nix-overlays";
-
-
-  outputs = { self, nixpkgs }:
-    let
-      pkgs = nixpkg.packages.${"YOUR_SYSTEM_STRING"};
-    in
-    {
-      ...
-    };
+  inputs.nixpkgs.url = "github:nix-ocaml/nix-overlays";
+
+  outputs = {
+    self,
+    nixpkgs,
+  }: let
+    # Include supported systems e.g ["x86_64-linux" "aarch64-darwin"]
+    supportedSystems = [];
+
+    forAllSystems = f:
```

Comment:
This just doesn't read better than `flake-utils`. The readme should cater to beginners who won't know what `lib` is, let alone `genAttrs`.

I like the complete example, but I'd appreciate if we could show `flake-utils` instead.

Reply:
And perhaps this example could be copied to a new section at the end of the readme that could serve as a reference on how to use the overlays without `flake-utils`.